keylime-ansible-gcp/README.md

```diff
@@ -0,0 +1,53 @@
+# Ansible Keylime for Google Cloud
+Ansible role to deploy a Fedora 35 instance on the Google Cloud Platform with [Keylime](https://github.com/keylime/keylime) and the [rust agent](https://github.com/keylime/rust-keylime) against a Virtualized TPM.
+
+Contributions are welcome, should anyone wish to have this role provision other Linux distributions.
+
+For details on using Keylime, please consult the
+[project documentation](https://keylime-docs.readthedocs.io/en/latest/)
+
+For details on the Rust agent, please consult the [repository](https://github.com/keylime/rust-keylime).
+
+## Configuration
+1.  Install dependecies for the Ansible - GCP module 
+`$ pip3 install requests google-auth`
+2. [Create a GCP service account](https://developers.google.com/identity/protocols/oauth2/service-account#creatinganaccount) for ansible's use.
+3. [Create and download keys linked to this service account](https://support.google.com/cloud/answer/6158849?hl=en&ref_topic=6262490#serviceaccounts&zippy=%2Cservice-accounts)
+4. Add path to ssh key to ansible config (/etc/ansible/ansible.conf) \
+Note: the ssh private key is in the downloaded file. Extract it, place it in its own file, set adequate permissions, and add the path to this newly created file to the ansible config. \
+Example:\
+[defaults] \
+private_key_file = /home/user/my_key
+
+5. Set environment variables \
```

Comment:
After doing this multiple times, I strongly feel it should be a script that can be called with a command like `. set_env_vars.sh` ([ref](https://stackoverflow.com/questions/496702/can-a-shell-script-set-environment-variables-of-the-calling-shell))